indica/bayesblackbox.py

```diff
@@ -0,0 +1,143 @@
+from copy import deepcopy
+import warnings
+from flatdict import FlatDict
+import numpy as np
+from scipy.stats import uniform
+np.seterr(all="ignore")
+warnings.simplefilter("ignore", category=FutureWarning)
+
+def gaussian(x, mean, sigma):
```

Comment:
we have too many of gaussian() functions across the project...
profiles_gauss has a quite general version of it, including background, amplitude, width and centroid, but it's missing the normalisation that you have in this version...
Can we generalise it and put in just 1 place, e.g. utilities or physics?